packages/pinorama-studio/src/contexts/theme-context.tsx

```diff
@@ -20,6 +20,12 @@ const initialState: ThemeProviderState = {
 
 const ThemeProviderContext = createContext<ThemeProviderState>(initialState)
 
+export const THEME: Record<Uppercase<Theme>, Theme> = {
+  DARK: "dark",
+  LIGHT: "light",
+  SYSTEM: "system"
+}
```

Comment:
Could you please use an enum for THEME in pascal case? For example, Theme.Dark instead of DARK. Thanks!

Reply:
Using enums in TypeScript is considered a bad practice: https://m.youtube.com/watch?v=jjMbPt_H3RQ

Reply:
I changed the case as requested

Reply:
Wow, I didn't know! Thanks for sharing!